README.md

```
@@ -63,6 +63,7 @@ WIP
 
 WIP \
 Either use one of the pre-compiled apps for your Operating system or compile from source
+> Or, you can install the latest version with the AUR - https://aur.archlinux.org/packages/void-mod-manager-git
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)